frontend/src/components/template/TheNavBar.vue

```diff
@@ -67,6 +67,13 @@ export default {
     menuTabClass(route, routePath) {
       return route.path.startsWith(routePath) ? 'active' : 'inactive';
     },
+    hasPermission(permission) {
+      return this.$store.getters['auth/hasPermission'](permission)
+    },
+    hasEligibilityPermission() {
+      const hasPermission = this.$store.getters['auth/hasPermission']
+      return hasPermission('E45') || hasPermission('R15')
```

Comment:
Not code problem or something to address in this PR, just saying this to get us thinking about it:
We have a bit of a disconnect between the use of business/service names and transaction names in various places. With us mostly moving away from v2 transactions entirely, it might be good to work with the business to just standardize on using the business/service names everywhere.

Reply:
It does make sense to move away from the legacy transaction names. The only current relevance is for the downstream transactions that still use HL7v2 messages. We would need to revisit the both the permission groups (e.g. E45 is a current SPG) and the transactions within. I created https://dev.azure.com/cyrovalente/HNWeb/_backlogs/backlog/HNWeb%20Team/Backlog%20items/?workitem=5343 to track this.